src/main/setup.hpp

```diff
@@ -4,8 +4,15 @@
 #include "SDL_video.h"
 const static int SDL_FLAGS = SDL_SWSURFACE | SDL_DOUBLEBUF;
 #endif
+#ifndef __CELLOS_LV2__
 const static char* FILENAME_CONFIG = "./config.xml";
 const static char* FILENAME_SCORES = "./hiscores";
 const static char* FILENAME_TTRIAL = "./hiscores_timetrial";
 const static char* FILENAME_CONT   = "./hiscores_continuous";
+#else
```

Comment:
Just for the sake of completeness, on the ps3, retroarch is installed always in `/dev_hdd0/game/SSNE10000/USRDIR/` and the user can only select folders inside it....so this system is pratically fixed about folders.